geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+
+# Rcm: Distance between the center-of-mass of CO subunit and the center-of-mass of CF3I subunit. 
+# ϕ: Angle between the C3 axis of CF3I subunit and Rcm.
+# φ: Angle between the OC axis and Rcm
+
+NAME = C2HAl 2012
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: a. Sun M, Halfen DT, Min J, Clouthier DJ, Ziurys LM (2012) Gas-phase rotational spectroscopy of AlCCH(X1Σ+): 
+A model system for organo-aluminum compounds. Chem Phys Lett 553:11-16 b. Mück LA, Thorwirth S, Gauss J (2015) The semiexperimental equilibrium structures of AlCCH and AlNC. J Mol Spectrosc 311:49-53.
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Al   C 1.963(5)
+C  ≡ C 1.210(7)
+C    H 1.060(3)
+
+# Bond lengths: rs (angstroms)
+
+Al   C 1.978
+C  ≡ C 1.202 
+C    H 1.060
+
+# Bond lengths: rm^(1) (angstroms)
+
+Al   C 1.986(1)
+C  ≡ C 1.2061(6)
+C    H 1.0634(3)
+
+# Bond lengths: re^se (angstroms)
+
+Al   C 1.9570(1)
+C  ≡ C 1.2223(2)
+C    H 1.0650(1)
+
+NAME = C2HAu 2013
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Okabayashi T, Kubota H, Araki M, Kuze N (2013) Microwave spectroscopy of AgCCH and AuCCH in the X1Σ+ states. Chem Phys Lett 577:11-15
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Au   C 1.903
+C  ≡ C 1.212
+C    H 1.055
+
+# Bond length C ≡ C 1.212 assumed at the value for CuCCH
+
+NAME = C2HCr
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Min J, Ziurys LM (2016) Millimeter-wave spectroscopy of CrC (X3Σ-) and CrCCH (X6Σ+): Examining the
+chromium-carbon bond. J Chem Phys 144(18):184308/1-184308/8
+[http://dx.doi.org/10.1063/1.4947247]
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Cr   C 1.993(6)
+C  ≡ C 1.238
+C    H 1.048
+
+# Bond length C ≡ C 1.238 and C H 1.048 assumed equal to that in ZnCCH
+
+NAME = C2HCu
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Sun M, Halfen DT, Min J, Harris B, Clouthier DJ, Ziurys LM (2010) The rotational spectrum of CuCCH (X1Σ+): A Fourier transform microwave discharge assisted laser ablation spectroscopy and millimeter/submillimeter
+study. J Chem Phys 133(17):174301/1-174301/8
+doi:10.1063/1.3493690
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Cu   C 1.818(1)
+C  ≡ C 1.212(2)
+C    H 1.058(1)
+
+# Bond lengths: rs (angstroms)
+
+Cu   C 1.819
+C  ≡ C 1.213
+C    H 1.058 
+
+# Bond lengths: rm^(1) (angstroms)
+
+Cu   C 1.8177(6)
+C  ≡ C 1.2174(6)
+C    H 1.046(2)
+
+NAME = C2HNO 2012
+# Point Group: Coov 
+# Shape: Linear --- Linear
+# Bond lengths: re (angstroms)
+# Source paper: McElmurry BA, Rivera-Rivera LA, Scott KW, Wang Z, Leonov II, Lucchese RR, Bevan JW (2012) Studies of
+low-frequency intermolecular hydrogen-bonded vibrations using a continuous supersonic slit jet mid-infrared
+quantum cascade laser spectrometer. Chem Phys 409:1-10
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+OC1    HCN 4.842(1)
+C(1)...H   2.572(2)
```

Comment:
Parentheses are probably not needed for C(1). C1 is probably just as good. I saw this before, but didn't comment on it. If the parentheses can be removed universally without me finding all previous examples, I would appreciate it!

Reply:
Fixed

Reply:
OC_1, OC_2, OC_3, ... 
or
OCa, OCb, OCc, ...
would require even fewer characters than:
OC(1), OC(2), OC(3), ...
But we can fix that later quite easily when we decide on a notation.